packages/persona/test/getPersonasByPersonaId.test.ts

```
@@ -1,28 +1,74 @@
 import { describe, expect, it } from "vitest";
-import { PersonaRecord } from "rusty-motors-shared";
 import { getPersonasByPersonaId } from "../src/getPersonasByPersonaId.js";
+import { PersonaRecord } from "../src/PersonaMapsMessage.js";
 
 describe("getPersonasByPersonaId", () => {
 	it("returns a persona", async () => {
 		// arrange
-		const personaRecord: PersonaRecord = {
-			id: Buffer.alloc(4),
-			name: Buffer.from("test"),
+		const personaRecord: Pick<
```

Biome

[error] This variable is unused.

Unused variables usually are result of incomplete refactoring, typos and other source of bugs.
Unsafe fix: If this is intentional, prepend personaRecord with an underscore.


(lint/correctness/noUnusedVariables)